packages/wonder-blocks-form/src/components/checkbox-core.tsx

```diff
@@ -6,17 +6,35 @@ import {addStyle} from "@khanacademy/wonder-blocks-core";
 import Icon from "@khanacademy/wonder-blocks-icon";
 
 import type {IconAsset} from "@khanacademy/wonder-blocks-icon";
-
 import type {ChoiceCoreProps} from "../util/types";
 
+type Checked = boolean | null | undefined;
+
+type AriaChecked = "true" | "false" | "mixed";
+
+function mapCheckedToAriaChecked(value: Checked): AriaChecked {
```

Comment:
**suggestion:** It would be also helpful adding comments here about the why we define this intermediate step... or describing the purpose of this function (e.g. expressing that we convert the prop value to a value that a Screen Reader can understand via the `aria-checked` attribute).